### DISCUSSION TITLE
What is content: unknown ?

### DISCUSSION THREAD

**Discussion by fmiqbal** [provectus/kafka-ui#3011] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8409391/205249377-523f1b3d-3e1b-47d5-bdd0-3c676c8b2fda.png)

What is this content unknown, key unknown ? does it serve any purpose ?

**Comment by Mgrdich** [comment]
it is related to the message interface where the type if it is being send from the BE 

```
/**
     * 
     * @type {MessageFormat}
     * @memberof TopicMessage
     */
    keyFormat?: MessageFormat;
    /**
     * 
     * @type {MessageFormat}
     * @memberof TopicMessage
     */
    valueFormat?: MessageFormat;
```
```
/**
 * 
 * @export
 * @enum {string}
 */
export enum MessageFormat {
    AVRO = 'AVRO',
    JSON = 'JSON',
    PROTOBUF = 'PROTOBUF',
    UNKNOWN = 'UNKNOWN'
}
```

**Comment by Haarolean** [comment, marked as answer]
@fmiqbal hey, it's kinda outdated. We'll display key/value serdes for each message soon (#3117)
